### DISCUSSION TITLE
Vagas de desenvolvedor no sul do país

### DISCUSSION THREAD

**Discussion by Murillofilho86** [frontendbr/forum#1004] [Discussão]
Sou programador c# e tenho interesse em trabalhar no sul do Brasil, mas o mercado de trabalho é bom?

**Comment by heliomsolivas** [comment]
Eu também tenho interesse em ir pra lá 👍

**Comment by kivervinicius** [comment]
somos 3 :)

**Comment by klarkc** [comment]
Sem dúvidas, estou trabalhando em floripa a quase um ano, pra mim não tem lugar melhor para ser dev do que aqui. Chove vagas o tempo todo, se você souber mesmo, tem trabalho garantido. Outro lugar q também se fala muito bem é porto alegre. Acho que para backend ainda seja melhor.

**Comment by Murillofilho86** [comment]
@klarkc Florianópolis é uma boa... tu sabe qual é a linguagem mais pedida?
